.build-linux/publishViewsInSchema.ps1

```diff
@@ -132,8 +131,9 @@ ForEach-Object {
     $nuspec = '<?xml version="1.0"?><package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd"><metadata>'
     $nuspec += "<id>$packageId</id>"
     $nuspec += "<version>$version$metadata</version>"
-    $nuspec += "<authors>$machineName</authors>"
+    $nuspec += "<authors>$packageId</authors>"
     $nuspec += "<description>Table version of the $packageId view.</description>"
+    $nuspec += "<repository url=`"$RepositoryUrl`"></repository>"
```

Comment:
I asked this on your other PR but will this behave if the repository url is empty?

Reply:
I'll answer here too for posterity:

Nothing seems to happen other than the package isn't associated with a repository. If you enter a bad url the same thing happens. I did notice some odd behavior in my tests that makes me think if packages of the same name already exist and you mess up the repository url then the package still acts like the previous packages before it (i.e. will be associated with a repository or not depending on whether previous packages were). I assume that's some GitHub black magic behind the scenes.